docs/Testing.md

```diff
@@ -0,0 +1,16 @@
+# Testing
+
+## Integration tests
+
+To run the integration tests, run `make test` in the root.
+This will create a backwards-compatible build.
```

Comment:
What does "backwards-compatible" mean here? Compatible with what?

Reply:
Compatible with a not-JS-Module, I suppose?

Reply:
Does `make test` overwrite build artifacts other than the HTML files?

Reply:
If not, I think it might be clearer to say something like "This will generate HTML files in `test/`."

Reply:
Not the ones we consume. It overwrites `mathquill-basic.js`. The only one that is consumed through the package is the minified version of `mathquill.js`, and that one is unaffected.